src/solver/simulation/common-hydro-remix.cpp

```diff
@@ -284,7 +287,8 @@ std::shared_ptr<IStorageForRemix> extractHydroForRemix(const Data::Area& area,
                              initLevel,
                              capacity,
                              efficiency,
-                             reservoirManagement);
+                             reservoirManagement,
+                             std::string(area.name) + " hydro");
```

Comment:
Appending "hydro" to storage name could be done inside **makeHydroForRemix**, this would avoid the caller to do it, make this caller a bit more simple.